README.md

```diff
@@ -21,11 +21,13 @@ JSON is a popular encoding format for geospatial data. The light weight, simple
 
 OGC Features and Geometries JSON will
 
-* include the ability to use Coordinate Reference Systems (CRSs) other than WGS84,
+* include the ability to use Coordinate Reference Systems (CRSs) other than WGS84 
+* follow the https://portal.opengeospatial.org/files/?artifact_id=76024[OGC Axis Order Policy] when CRSs other than WGS84 are used,
```

Comment:
I think "when CRSs other than WGS84 are used" can be omitted. The Axis Order Policy applies also for the default CRS.